angular/projects/global/src/lib/services/logger.service.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable, isDevMode } from '@angular/core';
```

Comment:
Sure looks familiar... but easily one of the most perfect and beautiful pieces of code I've ever seen in my life.

![image](https://user-images.githubusercontent.com/13532290/92952955-af413c00-f415-11ea-8911-26fbe4bc03d9.png)